[2023-01-28 01:07:24,948] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-06-01T00:00:00+00:00 [queued]>
[2023-01-28 01:07:24,958] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-06-01T00:00:00+00:00 [queued]>
[2023-01-28 01:07:24,959] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-01-28 01:07:24,960] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2023-01-28 01:07:24,961] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-01-28 01:07:24,973] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-06-01 00:00:00+00:00
[2023-01-28 01:07:24,981] {standard_task_runner.py:52} INFO - Started process 2200 to run task
[2023-01-28 01:07:24,984] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'local_to_gcs_task', 'scheduled__2019-06-01T00:00:00+00:00', '--job-id', '92', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmp2gc0y4u9', '--error-file', '/tmp/tmp28ftbm76']
[2023-01-28 01:07:24,986] {standard_task_runner.py:77} INFO - Job 92: Subtask local_to_gcs_task
[2023-01-28 01:07:25,049] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-06-01T00:00:00+00:00 [running]> on host adbf820de9e6
[2023-01-28 01:07:25,095] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2023-01-28 01:07:25,116] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-06-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-06-01T00:00:00+00:00
[2023-01-28 01:08:58,727] {python.py:175} INFO - Done. Returned value was: None
[2023-01-28 01:08:58,741] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20190601T000000, start_date=20230128T010724, end_date=20230128T010858
[2023-01-28 01:08:58,798] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-01-28 01:08:58,837] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-01-29 12:43:44,435] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-06-01T00:00:00+00:00 [queued]>
[2023-01-29 12:43:44,451] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-06-01T00:00:00+00:00 [queued]>
[2023-01-29 12:43:44,453] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-01-29 12:43:44,454] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2023-01-29 12:43:44,455] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-01-29 12:43:44,466] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-06-01 00:00:00+00:00
[2023-01-29 12:43:44,471] {standard_task_runner.py:52} INFO - Started process 7012 to run task
[2023-01-29 12:43:44,475] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'local_to_gcs_task', 'scheduled__2019-06-01T00:00:00+00:00', '--job-id', '401', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpty72ry_4', '--error-file', '/tmp/tmppfjvpilr']
[2023-01-29 12:43:44,477] {standard_task_runner.py:77} INFO - Job 401: Subtask local_to_gcs_task
[2023-01-29 12:43:44,539] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-06-01T00:00:00+00:00 [running]> on host 2530f37057aa
[2023-01-29 12:43:44,584] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2023-01-29 12:43:44,614] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-06-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-06-01T00:00:00+00:00
[2023-01-29 12:43:57,279] {python.py:175} INFO - Done. Returned value was: None
[2023-01-29 12:43:57,294] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20190601T000000, start_date=20230129T124344, end_date=20230129T124357
[2023-01-29 12:43:57,348] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-01-29 12:43:57,409] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
